tolgee-action/src/utils.ts

```diff
@@ -0,0 +1,94 @@
+import * as core from '@actions/core'
+import { exec } from '@actions/exec'
+import { HttpClient } from '@actions/http-client'
+import { mkdirP, which } from '@actions/io'
+import { rm } from '@actions/io/lib/io-util'
+import * as fs from 'fs'
+import { LanguageResponse } from './types'
+
+const base_url = 'https://app.tolgee.io/v2'
+const languages_url = `${ base_url }/projects/languages`
+const export_url = `${ base_url }/projects/export?format=JSON&filterState=TRANSLATED%2CREVIEWED%2CDISABLED`
+
+async function extractProjectLanguages(httpClient: HttpClient, api_key: string) {
```

Comment:
Can't we just to the curl step like we did before? 🤔

Reply:
It just prevents having to save a file and stuff, we directly extract json from response and process it to get languages metadata

Reply:
This function only downloads the languages metadata, real translations are downloaded later on in `exportLocaleData()`